### PR TITLE
Add leaf60 and tada68 keymaps

### PR DESCRIPTION
**Pull request by JarredSteenvoorden**
## Description

Add my leaf60 and tada68 keymaps

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/foxlab/leaf60/universal/keymaps/jarred/keymap.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "jarred.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_QW] = LAYOUT_60_hhkb(
+    KC_ESC , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSLS, KC_GRV , \
```
Comment (fauxpark) [inline keyboards/foxlab/leaf60/universal/keymaps/jarred/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/tada68/keymaps/jarred/rules.mk [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Build Options
```
Comment (fauxpark) [inline keyboards/tada68/keymaps/jarred/rules.mk]: This can be condensed down to just:

```make
MOUSEKEY_ENABLE = yes
CONSOLE_ENABLE = no
```

**Comment by JarredSteenvoorden** [comment]
Thanks :) have updated the files

File: keyboards/tada68/keymaps/jarred/keymap.c [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ */
+#include QMK_KEYBOARD_H
+
+#define _QW 0
+#define _NV 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_QW] = LAYOUT_ansi(
+    KC_ESC , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_GRV , \
```
Comment (fauxpark) [inline keyboards/tada68/keymaps/jarred/keymap.c]: Here, too ;)

**Pull request merged**